packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -8,6 +8,18 @@ import 'package:flutter/painting.dart';
 
 export 'dart:ui' show TextDirection;
 
+/// Determines the politeness setting of aria live in Flutter web.
+///
+/// It is used to set the priority with which screen reader should treat updates to live regions.
+enum AriaLivePolitenessSetting {
+  /// The screen reader will speak changes whenever the user is idle.
```

Comment:
nit: I believe we refer to screen readers as "assistive technology" because not all users use screen readers for accessibility. Also, announcements are frequently conveyed through audio, and therefore screen reader is not involved.